### PR TITLE
[WIP] MangoHud: update to 0.6.8.

### PR DESCRIPTION
**Pull request by 241m**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

File: srcpkgs/MangoHud/template [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Template file for 'MangoHud'
 pkgname=MangoHud
-version=0.6.7.1
-_version="${version%.*}-${version##*.}"
-revision=2
+version=0.6.8
+#_version="${version%.*}-${version##*.}"
+_version="${version}"
```
Comment (paper42) [inline srcpkgs/MangoHud/template:5]: drop $_version and remove the comment, it doesn't make sense anymore

**Comment by classabbyamp** [comment]
> - [Allow use of system nlohmann_json dependency](https://github.com/flightlessmango/MangoHud/commit/e1ffa0f85820abea44639438fca2152290c87ee8)

add `json-c++` to makedepends

**Comment by 241m** [comment]
> > * [Allow use of system nlohmann_json dependency](https://github.com/flightlessmango/MangoHud/commit/e1ffa0f85820abea44639438fca2152290c87ee8)
> 
> add `json-c++` to makedepends

If I'm not mistaken, the nlohmann_json dependency is only used by mangoapp ([src/meson.build](https://github.com/flightlessmango/MangoHud/blob/efdcc6d2f54e37a3a32475453407f1eb33d1bef2/src/meson.build#L248)), and AFAICT, is not built, nor installed by default ([meson_options.txt](https://github.com/flightlessmango/MangoHud/blob/efdcc6d2f54e37a3a32475453407f1eb33d1bef2/meson_options.txt#L17)).

I could be wrong, as this is my first time working with meson. If not, maybe a mangoapp subpackage should be added instead? Ala its [AUR](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=mangohud) counterpart?

Anyway, I'm marking this with WIP for now.

**Comment by onli** [comment]
I was looking into the state of mangohud - was the task of this PR solved somewhere else? Package seems to sit at 0.6.7.1 and this upgrade missed?

**Pull request closed without merging**